rastervision/command/train_command_config.py

```diff
@@ -92,6 +92,7 @@ def with_experiment(self, experiment_config):
         return b
 
     def with_task(self, task):
+        """Sets a specific task type, e.g. rv.OBJECT_DETECTION."""
```

Comment:
This makes it seem like you can pass rv.OBJECT_DETECTION, but you actually need to pass a TaskConfig object. same comment applies in other places.

Reply:
Done.